<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>window_gerenciarUsuarios</class>
 <widget class="QDialog" name="window_gerenciarUsuarios">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>676</width>
    <height>413</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gerenciar Usuarios</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>641</width>
     <height>391</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab_novoUsuario">
    <attribute name="title">
     <string>Novo Usuário</string>
    </attribute>
    <layout class="QVBoxLayout" name="verticalLayout_6">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <property name="rightMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Nome</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="txt_nomeNovoUsuario"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Username</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="txt_usernameNovoUsuario"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>telefone</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="txt_telefoneNovoUsuario"/>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Senha</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="txt_senhaNovoUsurio"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item>
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Tipo Acesso</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="cb_acessoNovoUsuario"/>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="btn_novoUsuario">
         <property name="text">
          <string>Limpar Campos</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_gravaNovoUsuario">
         <property name="text">
          <string>Gravar</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="tab_gerenciarUsuarios">
    <attribute name="title">
     <string>Gerenciar Usuários</string>
    </attribute>
    <widget class="QTableWidget" name="tw_geUsuarios">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>90</y>
       <width>321</width>
       <height>261</height>
      </rect>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>0</y>
       <width>120</width>
       <height>80</height>
      </rect>
     </property>
     <property name="title">
      <string>Filtro</string>
     </property>
     <widget class="QRadioButton" name="rb_geFiltroIdUsuario">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>100</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Id Usuário</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
      <property name="autoRepeat">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QRadioButton" name="rb_geFiltroNomeUsuario">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>100</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Nome</string>
      </property>
     </widget>
    </widget>
    <widget class="QPushButton" name="btn_geBuscarUsuario">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>60</y>
       <width>171</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Buscar</string>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>256</width>
       <height>341</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_13">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_7">
        <item>
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>Nome</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="txt_geNomeUsuario"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_8">
        <item>
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>Username</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="txt_geUserName"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_9">
        <item>
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>Senha</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="txt_geSenhaUsuario"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_11">
        <item>
         <widget class="QLabel" name="label_10">
          <property name="text">
           <string>Telefone</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="txt_geTelefoneUsuario"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_12">
        <item>
         <widget class="QLabel" name="label_11">
          <property name="text">
           <string>Acesso</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="cb_geAcessoUsuario"/>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer_3">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QPushButton" name="bnt_geGravarUsuario">
          <property name="text">
           <string>Gravar</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btn_geExcluirUsuario">
          <property name="text">
           <string>Excluir</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btn_geVendasUsuario">
          <property name="text">
           <string>Vendas</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>446</x>
       <y>10</y>
       <width>171</width>
       <height>52</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_10">
      <item>
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>Pesquisa</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="txt_gePesquisarUsuario"/>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
